bedrock/mozorg/templates/mozorg/contribute/signup-alternate.html

```diff
@@ -0,0 +1,123 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/contribute-base.html" %}
+
+{% add_lang_files "mozorg/contribute/index" %}
+
+{% block body_id %}inquiry{% endblock %}
+
+{% block page_title %}{{_('Get Involved: Volunteer Opportunities at Mozilla')}}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'contribute-signup' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if LANG == 'en-US' and waffle.switch(‘contribute-signup-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block contrib_nav_cta %}{% endblock %}
+{% block contrib_country_form %}{% endblock %}
+
+{% block contrib_title %}
+  <div class="container">
+    <h1 class="page-title">{{ _('Get Involved with Mozilla Today') }}</h1>
+  </div>
+{% endblock %}
+
+
+{% block contrib_content %}
+<article>
+
+  <section>
+    <header>
+      <h2>{{ _('Try something simple') }}</h2>
+    </header>
+
+    <ul class="simple">
+      <li class="task">
+        <h3><a href="{{ url('mozorg.contribute.taskview') }}whimsy/?v=2" class="do-task outline-button more">{{ _('Customize Your Desktop Browser') }}</a></h3>
```

Comment:
When browsing the page, I instinctively clicked on the image for each task. Though they have a hover state, they aren't active links. What do you think about making the image clickable as well?